docs/src/languages/php/_index.md

```diff
@@ -22,139 +22,127 @@ Note that from PHP 7.1, the images use the Zend Thread Safe (ZTS) version of PHP
-* On PHP 7.4, it's `/usr/sbin/php-fpm7.4-zts`.
+### 2. Specify a dependency manager
 
-While you can call it manually that's generally not necessary.
-Note that PHP-FPM can't run simultaneously along with another persistent process (such as ReactPHP or Amp).
-If you need both, they have to run in separate containers.
+To manage PHP dependencies and libraries, use [Composer](https://getcomposer.org/).
+PHP containers use Composer 1.x by default.
 
-## Expanded dependencies
+To use Composer 2.x on your project or for more options, see [dependency management](#dependencies).
 
-In addition to the standard `dependencies` format,
-it's also possible to specify alternative repositories for use by Composer.
-The standard format:
+### 3. Serve your app
 
-```yaml
-dependencies:
-    php:
-        "platformsh/client": "2.x-dev"
-```
+To serve your app, define what (and how) content should be served by [setting the `locations`](../../create-apps/app-reference.md#locations).
 
-is equivalent to `composer require platform/client 2.x-dev`.
-You can also specify explicit `require` and `repositories` blocks:
+Usually, it contains the two following (optional) keys:
+- A `root` directory (docroot) folder to which all requests for existing `.php` and static files (such as `.css`, `.jpg`) are sent.
+- A `passthru` to define a front controller to handle non-existent files.
+  The value is a file path relative to the [app root](../../create-apps/app-reference.md#root-directory).
 
-```yaml
-dependencies:
-    php:
-        require:
-            "platformsh/client": "2.x-dev"
-        repositories:
-            - type: vcs
-              url: "git@github.com:platformsh/platformsh-client-php.git"
+Adjust the `locations` block to your needs.
+
+In the following example, all requests made to your site's root (`/`) are sent to the `public` directory
+and nonexistent files are handled by `app.php`:
+
+```yaml {location=".platform.app.yaml"}
+web:
+    locations:
+        '/':
+            root: 'public'
+            passthru: '/app.php'
 ```
 
-That would install `platformsh/client` from the alternate repository specified, as a global dependency.
-In other words, it's equivalent to the following `composer.json` file:
-
-```json
-{
-    "repositories": [
-        {
-            "type": "vcs",
-            "url":  "git@github.com:platformsh/platformsh-client-php.git"
-        }
-    ],
-    "require": {
-        "platformsh/client": "2.x-dev"
-    }
-}
+See how to [create a basic PHP app with a front controller](../../create-apps/web/php-basic.md).
+To have more control, you can define rules to specify which files you want to allow [from which location](../../create-apps/web/php-basic.md#set-different-rules-for-specific-locations).
+
+### Complete example
+
+A complete basic app configuration looks like the following:
+
+```yaml {location=".platform.app.yaml"}
+name: 'app'
+
+type: 'php:8.1'
+
+disk: 2048
+
+hooks:
+    build: |
+        set -e
+
```

Comment:
:hammer_and_wrench: Given that this isn't addressed at all above and doesn't do anything.
```suggestion
```